docs/book/releases/v2.1.1.md

```diff
@@ -52,6 +52,11 @@
    - Impact: In-tree vSphere volumes will not get migrated successfully
    - Workaround:
       - Upgrade CSI driver with this fix.
+7. When pod using a PVC is rescheduled to other node when metadatasyncer is down, fullsync might fail with error `Duplicated entity for each entity type in one cluster is found`.
+   - Impact: CNS may hold stale volume metadata.
+   - Workaround:
+      - Unregister this volume with CNS and then register this volume with CNS.
```

Comment:
Same comment as above. Let's remove this as a workaround.

Reply:
Fixed.